modules/ROOT/nav.adoc

```diff
@@ -91,56 +91,58 @@ include::partial$nav-migration.adoc[]
   *** xref:arm-deploy-mule-application-mmp-reference.adoc[ARM Deployment Reference (Mule Maven Plugin)]
   *** xref:agent-deploy-mule-application-mmp-task.adoc[Deploy a Mule App Using the Mule Agent (Mule Maven Plugin)]
   *** xref:agent-deploy-mule-application-mmp-reference.adoc[Mule Agent Deployment Reference (Mule Maven Plugin)]
-* xref:dataweave.adoc[DataWeave Language]
- ** xref:dataweave-language-introduction.adoc[DataWeave Scripts]
- ** xref:dataweave-selectors.adoc[DataWeave Selectors]
- ** xref:dataweave-formats.adoc[Data Formats Supported by DataWeave]
-  *** xref:dataweave-flat-file-schemas.adoc[DataWeave Flatfile Schemas]
- ** xref:dataweave-types.adoc[DataWeave Types]
-  *** xref:dataweave-types-coercion.adoc[Type Coercion with DataWeave]
- ** xref:dataweave-variables.adoc[DataWeave Variables]
-  *** xref:dataweave-variables-context.adoc[DataWeave Variables for Mule Runtime]
- ** xref:dataweave-flow-control.adoc[Flow Control in DataWeave]
- ** xref:dataweave-pattern-matching.adoc[Pattern Matching in DataWeave]
+* xref:dataweave.adoc[DataWeave 2.0]
+ ** xref:dataweave-quickstart.adoc[DataWeave Tutorial]
+ ** xref:dataweave-language-guide.adoc[DataWeave Language Guide]
+  *** xref:dataweave-language-introduction.adoc[Scripts]
+  *** xref:dataweave-selectors.adoc[Selectors]
+  *** xref:dataweave-formats.adoc[Data Formats]
+  *** xref:dataweave-flat-file-schemas.adoc[Flatfile Schemas]
+  *** xref:dataweave-types.adoc[Types]
+  *** xref:dataweave-types-coercion.adoc[Type Coercion]
+  *** xref:dataweave-variables.adoc[Variables]
+  *** xref:dataweave-variables-context.adoc[Variables]
```

Comment:
"Variables" appears twice here. The second "Variables" should be closer to the title of `dataweave-variables-context.adoc`, such as  "Variables for Mule Runtime"

Reply:
Searching for a name that will disambiguate it from the "traditional" variables which users create. Unfortunately, "Variables for Mule Runtime" don't disambiguate them.

@dukesphere Here are some ideas, what do you think?
1. DW variables for Mule objects (but not always objects?)
2. DW Mule system variables? DW environment variables?
3. DW variables for Mule values (as opposed to data <mapping> values?)
4. DW variables for retrieving Mule object data?

Something better?